### ISSUE TITLE
[Feature] List incompatible mods

### ISSUE THREAD

**Issue by ChristopherHaws** [packwiz/packwiz#145]
It would be awesome if there were a command to list any mods that are not compatible with the current `acceptable-game-versions`. After updating to 1.19.1 I am trying to figure out which mods are still pending updates and the only easy way right now is for me to simply run the game and see which mods cause it to crash or go to each mod page manually.

Thanks!

**Comment by rhullah** [comment]
I agree, something like this would be helpful.  You kinda can figure it out by upgrading the MC version and then running `packwiz update --all` which will prompt you with a list of all the mods it'll update.  So anything it doesn't list would likely be incompatible.

**Comment by rhullah** [comment]
Further clarification: not only will the `--all` option list mods it'll upgrade, but it'll list the mods that (1) it can't update likely because it's a direct download url; and (2) failed to find a valid version.  The latter (2) are the mods that were not found to match the `acceptable-game-versions`.

You can then proceed with the update by pressing `Y` or cancel by pressing `n`.

**Comment by ChristopherHaws** [comment]
> `--all` option list mods it'll upgrade, but it'll list the mods that (1) it can't update likely because it's a direct download url; and (2) failed to find a valid version. The latter (2) are the mods that were not found to match the `acceptable-game-versions`.
> 
> You can then proceed with the update by pressing `Y` or cancel by pressing

Many mods support 1.19 and 1.19.1 so they have already been updated in my modpack (which supported 1.19 and I am updating to support 1.19.1). This means that running `packwiz update --all` doesn't list those mods even though they do support 1.19.1 because they have already been updated to a version that supports 1.19.1.

**Comment by rhullah** [comment]
Sure, I get that.  I thought you were looking for incompatible mods.  So I thought the mods that failed the `update --all` with "failed to find a valid version" might be sufficient.

**Comment by comp500** [comment]
See also: https://github.com/packwiz/packwiz/issues/40
